.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'docker' # See documentation for possible values
+    directory: '/db' # Location of package manifests
```

Comment:
only /db ? what about /app ?

Reply:
It's just on the next one since directory won't take an array, although if you check the discussion on teams we won't be using dependabot since we can't get the function we want out of it